programmers/20210514/크레인-인형뽑기-게임.js

```diff
@@ -0,0 +1,22 @@
+function solution(board, moves) {
+    const basket = [];
+    let result = 0
+    for (let i = 0; i < moves.length; i++) moves[i]--;
+    for (let move of moves) {
+        for (let select = 0; select < board.length; select++) {
+            if (board[select][move] !== 0) {
+                basket.push(board[select][move]);
+                board[select][move] = 0;
+                break;
+            }
+        }
+        if (basket.length > 1) {
+            if (basket[basket.length - 1] === basket[basket.length - 2]) {
+                basket.pop();
+                basket.pop();
```

Comment:
pop()을 두번 쓰는것보다 한줄로 처리하는 것이 더 깔끔합니다.
```suggestion
                basket.splice(basket.length - 2, 2);
```